lib/browser/api/menu.js

```diff
@@ -164,10 +164,15 @@ Menu.buildFromTemplate = function (template) {
   const sorted = sortTemplate(filtered)
 
   sorted.forEach((item) => {
-    if (typeof item !== 'object') {
+    if (typeof item !== ('object' || null)) {
       throw new TypeError('Invalid template for MenuItem')
     }
-    menu.append(new MenuItem(item))
+    if (Object.keys(item).length > 0 && item !== null) {
```

Comment:
I think this will crash if `item` is `null` because the first test will run first and `null` can't be converted to an object.